### PR TITLE
chore: increate nexttest slow timeout period

### PR DESCRIPTION
**Pull request by arminsabouri**

### CODE REVIEW

**Comment by arminsabouri** [comment]
TODO : increase number of nexttest threads to 8

File: bin/test-suite/src/suite/consensus/common/poa_node.rs [hunk 1]
```diff
@@ -441,13 +441,13 @@ impl PoaNodeCommandConfig for FederationMemberTestConfig {
...
                             let s = SigningStatus::try_from(status.status).ok();
                             if let Some(status) = s {
-                                let _ = rx_sender
-                                    .send(Notifications::SigningStatusReport((
-                                        engine_index,
-                                        session_id,
-                                        status,
-                                    )))
-                                    .expect("Failed to send signing status report");
+                                let _ = rx_sender.send(Notifications::SigningStatusReport((
```
Comment (BatiGencho) [inline bin/test-suite/src/suite/consensus/common/poa_node.rs]: check my PR where I am handling that similarly but with some relaxation

**Review by BatiGencho — DISMISSED** [review]

File: bin/test-suite/src/suite/consensus/common/poa_node.rs (hunk 1 above)
Comment (arminsabouri) [reply bin/test-suite/src/suite/consensus/common/poa_node.rs]: Ended up rebasing your changes in

**Review by scottmillner — DISMISSED** [review]

**Review by BatiGencho — DISMISSED** [review]

**Comment by arminsabouri** [comment]
Screenshot 2024-06-20 at 7 22 01 PM

Testing results ran locally

**Pull request merged**